tests/Resources/ti.ui.picker.addontest.js

```diff
@@ -0,0 +1,74 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2011-2018 by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+var should = require('./utilities/assertions');
+
+describe('Titanium.UI.Picker', function () {
+
+	var win;
```

Comment:
might want to "clean up" (close) the window after the test like the other UI test suites do.